utils/_context/_scenarios/__init__.py

```diff
@@ -333,6 +333,17 @@ class _Scenarios:
         scenario_groups=[ScenarioGroup.APPSEC],
     )
 
+    appsec_standalone_v2 = EndToEndScenario(
```

Comment:
For my knowledge, why not using `appsec_standalone` ?

Reply:
Seen on zoom, go for these 3 new scenario. Could you add them in `.github/workflows/end-to-end.yml`